prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -118,7 +118,7 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
     let enrolment = this.enrolmentService.enrolment;
 
     // Store current status as it will be truncated for initial enrolment
-    this.currentStatus = enrolment.currentStatus.statusCode;
+    this.currentStatus = enrolment?.currentStatus.statusCode;
```

Comment:
Feels like we want to use the mock enrolment service to ensure an enrolment exists instead of doing a null check since in the application the enrolment must exist, and if it didn't things are going sideways and we should let the application crash.